app/forms/validators.py

```diff
@@ -468,7 +468,13 @@ def __call__(
         is_mandatory: bool,
         is_only_checkboxes_or_radios: bool,
     ) -> None:
-        total_answered = sum(1 for value in answer_values if value)
+
+        empty_answer_values: tuple = (None, [], "", {})
```

Comment:
We should reuse the `EMPTY_ANSWER_VALUES` from the QuestionnaireStoreUpdater class